webui/react/src/fixtures

```diff
@@ -0,0 +1 @@
+../test/fixtures
```

Comment:
**suggestion:** this seems brittle to have it setup like this. What if we just had the configs in the react root directory `/webui/react/fixtures`?

Reply:
we would still need the symbolic link if it's not in the src directory because of CRA I think right? react root works.

Reply:
ah good question about CRA, yeah if CRA needs in in `src` so be it

Reply:
react/fixtures or /react/tests/fixtures?

Reply:
or just src/fixtures with no symlink

Reply:
we can always move it later if we don't like the position, but `src/fixtures` with no symlink sounds good